autoscheduler/frontend/src/components/HelpText.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { Grid, Paper } from '@material-ui/core';
+
+const HelpText: React.SFC = function App() {
+  return (
+    <Grid
+      container
+      spacing={0}
+      direction="row"
+      justify="space-around"
+      alignItems="center"
+      style={{ minHeight: '65vh' }}
+    >
+      <Paper style={{ width: '55%' }}>
```

Comment:
We might want to set a max width here, something like 800px so it doesn't get too wide on wide monitors

Reply:
Would using `width: '55%' || '800px'` be the right implementation?

Reply:
No, because JS wouldevaluate the first string as true and never look at the second string. Instead, set width to 55% and set a separate CSS property `maxWidth: '800px'`